app/grandchallenge/cases/models.py

```diff
@@ -846,6 +852,64 @@ class DICOMImageSetUploadStatusChoices(models.TextChoices):
     INITIALIZED = "INITIALIZED", _("Initialized")
     STARTED = "STARTED", _("Started")
     FAILED = "FAILED", _("Failed")
+    COMPLETED = "COMPLETED", _("Completed")
+
+
+class HealthImagingWrapper:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._health_imaging_client = boto3.client(
+            "medical-imaging",
+            region_name=settings.AWS_DEFAULT_REGION,
+        )
+
+    def start_dicom_import_job(self, *, job_name, input_s3_uri, output_s3_uri):
+        """
+        Start a HealthImaging DICOM import job.
+        """
+        return self._health_imaging_client.start_dicom_import_job(
+            jobName=job_name,
+            datastoreId=settings.AWS_HEALTH_IMAGING_DATASTORE_ID,
+            dataAccessRoleArn=settings.AWS_HEALTH_IMAGING_IMPORT_ROLE_ARN,
+            inputS3Uri=input_s3_uri,
+            outputS3Uri=output_s3_uri,
+        )
+
+    def delete_image_set(self, *, image_set_id):
+        try:
+            return self._health_imaging_client.delete_image_set(
+                imageSetId=image_set_id,
+                datastoreId=settings.AWS_HEALTH_IMAGING_DATASTORE_ID,
+            )
+        except self._health_imaging_client.exceptions.ThrottlingException as e:
+            raise RetryStep("Request throttled") from e
+        except ClientError as err:
+            logger.error(
+                "Couldn't delete image set. Here's why: %s: %s",
```

Comment:
Adding the code and message to the overall report will create unique errors in sentry. It is better to do:

```python3
except ClientError:
  logger.error("Couldn't delete image set", exc_info=True)
```

Then all of the information, including the exception, will be in Sentry and grouped appropriately.